lib/plugins/aws/package/compile/events/apiGateway/lib/method/index.js

```diff
@@ -14,7 +14,13 @@ module.exports = {
       const requestParameters = {};
       if (event.http.request && event.http.request.parameters) {
         Object.entries(event.http.request.parameters).forEach(([key, value]) => {
-          requestParameters[key] = value.required === undefined ? value : value.required;
+          /* eslint-disable no-nested-ternary */
```

Comment:
Let's not disable lint. Technically this rule is turned on, on purpose, so code style like that is not introduced :)

I know I've written comment abusing that rule, but it was just for quick reference.

We can write it differently, probably with IIFE:

```javascript
requestParameters[key] = (() => {
  if (!_.isObject(value)) return value;
  return value.required != null ? value.required : true
})();
```
```